CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 * `Menu` now outputs semantic HTML. Links are rendered in HTML lists, with submenus rendered with nested lists.
 
+## Bug Fixes
+
+* When selecting all in selectable grids with a secondary theme, the odd lined items were displaying a white background showing as being broken.
```

Comment:
`table` instead of `grid` - `grid` is a word we use at Sage that's not common and the component is called `Table`

The usual format is: `CompName: details of bug fix / improvement`